one/src/daemon.rs

```diff
@@ -495,18 +523,50 @@ pub async fn run(opts: DaemonOpts) -> Result<()> {
     let handle = signals.handle();
 
     // Start Flight server
-    let flight_handle = if let Some(addr) = opts.flight_sql_bind_address {
+    let (flight_handle, aggregator_handle) = if let Some(addr) = opts.flight_sql_bind_address {
         let addr = addr.parse()?;
         let feed = event_svc.clone();
-        let mut shutdown_signal = shutdown_signal.resubscribe();
-        Some(tokio::spawn(async move {
-            ceramic_flight::server::run(feed, addr, async move {
-                let _ = shutdown_signal.recv().await;
+        let bucket = opts
+            .s3_bucket
+            .ok_or_else(|| anyhow!("s3_bucket option is required when exposing flight sql"))?;
```

Comment:
*should* this be required?  What if you just want the run the aggregation locally and query out the aggregated state feed?

Reply:
Today its required as there is no mechanism for storing the state locally. We should likely add that for dev purposes but that is a separate issue.